CHANGELOG.md

```diff
@@ -1,12 +1,71 @@
 # osquery Changelog
 
+<a name="4.5.1"></a>
+## [4.5.1](https://github.com/osquery/osquery/releases/tag/4.5.1)
+
+[Git Commits](https://github.com/osquery/osquery/compare/4.5.0...4.5.1)
+
+
+### New Features
```

Comment:
`New Features` could maybe be a level 2 header as it seems the level 3s below are all associated with it?

Reply:
As it stands, All subheadings are 3 deep, and nested under the `4.5.1` tag. Previously, we had `## New Features / Under the Hood improvements` but it felt better split. There are no items in `New Features`

I suspect it looks more sensible in the whole view. and not the diff. But maybe there's something more clear